code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -26,9 +26,14 @@
 	item_state = "syringe_kit"
 	max_storage_space = DEFAULT_NORMAL_STORAGE
 	var/foldable = /obj/item/stack/material/cardboard	// BubbleWrap - if set, can be folded (when empty) into a sheet of cardboard
+	var/maxHealth = 20	//health is already defined
+	cant_hold = list(/obj/item/weapon/gun/projectile/automatic/sts35,
```

Comment:
Putting the sts35 will only block the addition of sts35 to the box, other automatics will go in.

Reply:
I dont think you understand, other automatics normally cannot fit into the box, or I was led to believe that. the sts issue has already been solved. I think its just shotguns now.